### PR TITLE
oopsy: change icons to fontawesome

### PR DESCRIPTION
**Pull request by quisquous**
I wanted to try to get back to the original flavor of (colored) win7 emoji with oopsy icons.  I find it very hard to parse the win10 colored emoji when I glance over sometimes.

Original win7 icons:
image

Current win10 icons: (death, fail, warn, pull, potion, wipe, good, damage, heal)
image

Proposed fontawesome icons:
image

Feel free to counterpropose from https://fontawesome.com/v5.15/icons?d=gallery&p=9&m=free or with some other license compatible svg icon.

### CODE REVIEW

**Comment by valarnin** [comment]
For heal, maybe:

image

For damage, maybe:

image

**Comment by quisquous** [comment]
image

image

Theoretically I like `first-aid` better, but I played with it some and in the live list with the glow-y shadow, `first-aid` loses some of its detail over `plus-square`.

`bomb` is nice too, but I worry that it's another circle, with both `ban` and `skull` being pretty circular too.  The two arrows (and the knife in the old version) are all much more distinctive shapes that they are easier to mentally parse, imo.  If fontawesome ever adds crossed swords / knife / etc, I think I'd be happy to switch to that.

**Comment by quisquous** [comment]
I think I'm gonna just land this as-is.  These are all overridable in user css, and so I mostly just want to provide good defaults.  Thanks for all the suggestions.
